percona-packaging/debian/changelog

```diff
@@ -1,5 +1,5 @@
-percona-server-mongodb-36 (3.6.0) UNRELEASED; urgency=medium
+percona-server-mongodb (4.0.3) UNRELEASED; urgency=medium
```

Comment:
The first GA release will be at least 4.0.4, not 4.0.3.

Reply:
It is ok as current version is 4.0.3.

Reply:
fixed